README.md

```diff
@@ -4,52 +4,65 @@
 [![All Contributors](https://img.shields.io/github/all-contributors/contentful/forma-36/master)](#contributors-)
 [![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lernajs.io/)
 
-This is the monorepo for the Forma 36 design system by [Contentful](https://www.contentful.com).
+[Forma 36](https://f36.contentful.com/) is an open-source design system by [Contentful](https://www.contentful.com) created with the intent to reduce the overhead of creating UI by providing tools and guidance for digital teams building and extending Contentful products
```

Comment:
Is this official copy? It's a bit convoluted.

Reply:
This bit is in our website, I just copied and put the links